src/openrct2-ui/windows/ServerList.cpp

```diff
@@ -98,7 +96,6 @@ namespace OpenRCT2::Ui::Windows
             SetWidgets(_serverListWidgets);
             widgets[WIDX_PLAYER_NAME_INPUT].string = const_cast<utf8*>(_playerName.c_str());
             InitScrollWidgets();
-            WindowSetResize(*this, { 320, 90 }, { 320, 90 });
```

Comment:
This line was removed because the function is called again on line 106/109, this time with named constants.